### ISSUE TITLE
No Name is announced for close, or post comment buttons present in fishbowl comment card by NVDA

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13495]
### Steps to reproduce:
Step 1: Open any Word app and open any document 
Step 2: Press Shift+F10 and move to new comment and activate it
Step 3: type something in fishbowl comment
Step 4: Navigate to the comment card in the comments pane and then navigate to the post comment and delete comment buttons
Step 5:  observe that there is no label announced for delete or post buttons in fishbowl comment card.

### Actual behavior:
No Name is announced for close, or post comment buttons present in fishbowl comment card.

### Expected behavior:
Name for each and every interactive control should be announced with NVDA.

### System configuration
OS Version: Win 11 Build (22489.1000)
Word Application Version: 2112 (Build 14715.20000)

#### NVDA installed/portable/running from source:
installed

#### NVDA version:
NVDA Version 2021.3.3

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.556)

#### Does the issue still occur after restarting your computer?
Yes

**Comment by michaelDCurran** [comment]
I cannot reproduce this. Perhaps this was fixed in a later build of Microsoft word?
Post comment and reply buttons are labelled correctly. Deleting a comment via the More comment actions or more thread actions menus is labelled correctly.
If you can still reproduce the issue yourself, please provide a an NVA log set to input/output, or video (with audio) showing the problem being reproduced.
